exporter/diags.go

```diff
@@ -71,21 +76,31 @@ type DiagnosticsCollector struct {
 	slurmTypeRpcTotalTime *prometheus.Desc
 	// daemon metrics
-	slurmCtlThreadCount    *prometheus.Desc
-	slurmDbdAgentQueueSize *prometheus.Desc
+	slurmCtlThreadCount            *prometheus.Desc
+	slurmDbdAgentQueueSize         *prometheus.Desc
+	slurmBackfillJobCount          *prometheus.Desc
+	slurmBackfillCycleCount        *prometheus.Desc
+	slurmBackfillLastDepth         *prometheus.Desc
+	slurmBackfillLastDepthTrySched *prometheus.Desc
+	slurmBackfillCycleCounter      *prometheus.Desc
 }
 
 func NewDiagsCollector(config *Config) *DiagnosticsCollector {
 	cliOpts := config.cliOpts
 	return &DiagnosticsCollector{
-		fetcher:                NewCliScraper(config.cliOpts.sdiag...),
-		slurmUserRpcCount:      prometheus.NewDesc("slurm_rpc_user_count", "slurm rpc count per user", []string{"user"}, nil),
-		slurmUserRpcTotalTime:  prometheus.NewDesc("slurm_rpc_user_total_time", "slurm rpc avg time per user", []string{"user"}, nil),
-		slurmTypeRpcCount:      prometheus.NewDesc("slurm_rpc_msg_type_count", "slurm rpc count per message type", []string{"type"}, nil),
-		slurmTypeRpcAvgTime:    prometheus.NewDesc("slurm_rpc_msg_type_avg_time", "slurm rpc total time consumed per message type", []string{"type"}, nil),
-		slurmTypeRpcTotalTime:  prometheus.NewDesc("slurm_rpc_msg_type_total_time", "slurm rpc avg time per message type", []string{"type"}, nil),
-		slurmCtlThreadCount:    prometheus.NewDesc("slurm_daemon_thread_count", "slurm daemon thread count", nil, nil),
-		slurmDbdAgentQueueSize: prometheus.NewDesc("slurm_dbd_agent_queue_size", "slurmDbd queue size. Number of threads interacting with SlrumDBD. Will grow rapidly if DB is down or under stress", nil, nil),
+		fetcher:                        NewCliScraper(config.cliOpts.sdiag...),
+		slurmUserRpcCount:              prometheus.NewDesc("slurm_rpc_user_count", "slurm rpc count per user", []string{"user"}, nil),
+		slurmUserRpcTotalTime:          prometheus.NewDesc("slurm_rpc_user_total_time", "slurm rpc avg time per user", []string{"user"}, nil),
+		slurmTypeRpcCount:              prometheus.NewDesc("slurm_rpc_msg_type_count", "slurm rpc count per message type", []string{"type"}, nil),
+		slurmTypeRpcAvgTime:            prometheus.NewDesc("slurm_rpc_msg_type_avg_time", "slurm rpc total time consumed per message type", []string{"type"}, nil),
+		slurmTypeRpcTotalTime:          prometheus.NewDesc("slurm_rpc_msg_type_total_time", "slurm rpc avg time per message type", []string{"type"}, nil),
+		slurmCtlThreadCount:            prometheus.NewDesc("slurm_daemon_thread_count", "slurm daemon thread count", nil, nil),
+		slurmDbdAgentQueueSize:         prometheus.NewDesc("slurm_dbd_agent_queue_size", "slurmDbd queue size. Number of threads interacting with SlrumDBD. Will grow rapidly if DB is down or under stress", nil, nil),
+		slurmBackfillJobCount:          prometheus.NewDesc("slurm_backfill_job_count", "slurm number of jobs started thanks to backfilling since last slurm start", nil, nil),
```

Comment:
```suggestion
		slurmBackfillJobCount:          prometheus.NewDesc("slurm_backfill_job_count", "Total backfilled jobs (since last slurm start)", nil, nil),
```

Reply:
I'd at least get rid of "thanks"